server/tests/message.integration.spec.js

```diff
@@ -389,21 +522,23 @@ describe('Message API:', function () {
                 .catch(done);
         });
 
-        xit('should delete the message from the DB', done => {
+        it('should archive message', done => {
             request(app)
-                .delete(baseURL + '/message/delete' + messageId)
+                .put(baseURL + '/message/archive/' + messageId)
+                .set('Authorization', `Bearer ${auth}`)
                 .expect(httpStatus.OK)
                 .then(res => {
                     let id = res.body.id;
-                    Message.findById(id)
-                        .then(message => {
-                            expect(message).to.be.null;
-                            done();
-                        });
+                    Message.unscoped()

```

Comment:
Would mean a change to deletion as well, but why do we need to unscope in the tests? Is it worth writing an object method to handle this for us? I like using scoping but would also like to provide a clean interface if possible.

Reply:
@jsachs, is there any other way we can handle this without unscoping first? `isDeleted: false` has already been included in the `defaultScope`.

Reply:
What I'm saying is, can we write a method or use a method so that the `unscope` call is hidden behind a nicer interface that just appears to get deleted/archived messages?